app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -27,151 +28,206 @@ function clusterName(clusters, clusterId) {
 const InfrastructureMappingsList = ({
   clusters,
   transformationMappings,
-  createInfraMappingClick
+  createInfraMappingClick,
+  inProgressRequestsTransformationMappings,
+  showDeleteConfirmationModalAction,
+  setMappingToDeleteAction,
+  showDeleteConfirmationModal,
+  hideDeleteConfirmationModalAction,
+  mappingToDelete,
+  yesToDeleteInfrastructureMappingAction,
+  notStartedTransformationPlans,
+  finishedWithErrorTransformationPlans
 }) => (
-  <Grid.Col
-    xs={12}
-    style={{
-      paddingBottom: 100,
-      height: '100%'
-    }}
-  >
-    {transformationMappings.length > 0 ? (
-      <React.Fragment>
-        <div className="heading-with-link-container">
-          <div className="pull-left">
-            <h3>{__('Infrastructure Mappings')}</h3>
-          </div>
-          <div className="pull-right">
-            {/** todo: create IconLink in patternfly-react * */}
-            <a
-              href="#"
-              onClick={e => {
-                e.preventDefault();
-                createInfraMappingClick();
-              }}
-            >
-              <Icon type="pf" name="add-circle-o" />
-              &nbsp;{__('Create Infrastructure Mapping')}
-            </a>
+  <div>
```

Comment:
i think we can just use `React.Fragment` instead of introducing the additional <div> ? i.e. just wrap <Grid.Col> and <DeleteInfrastructureMappingConfirmationModal> w/ <React.Fragment>